apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -267,7 +264,7 @@ private static String getClientSslPrincipalDistinguishedName(HttpServletRequest
   private static X509Certificate getClientCertificate(HttpServletRequest request) {
     X509Certificate[] certs =
         (X509Certificate[]) request.getAttribute("jakarta.servlet.request.X509Certificate");
-    if (certs == null || certs.length <= 0) {
+    if (certs == null || certs.length == 0) {
```

Comment:
Unrelated change here, but it irks me to see a `<= 0` check on a value that can only be positive.